setup.cfg

```diff
@@ -1,13 +1,36 @@
 [flake8]
 max-line-length = 110
-ignore = E133, E226, E228, N802, N803, N806, E266, N812, N815, N816, W503
+max-doc-length = 79
+ignore = E133, E226, E228, N802, N803, N806, N812, N815, N816, W503
 # TODO: remove E266 when Task documentation is converted to rst in DM-14207.
 exclude =
     __init__.py
+    assembleCcdTask.py E266 W505
```

Comment:
Please file a ticket to cover updating these files to be compliant, and add a comment here pointing to it.